package.json

```diff
@@ -62,7 +62,7 @@
     "classnames": "2.2.6",
     "common-tags": "1.8.0",
     "dom-helpers": "3.4.0",
-    "emotion": "9.2.12",
+    "@emotion/core": "10.0.1",
```

Comment:
Shouldn't we also install `@emotion/styled`?

https://emotion.sh/docs/install#with-styled